annotation-file-utilities/src/annotator/find/Insertion.java

```diff
@@ -50,22 +50,66 @@
     protected Set<String> packageNames;
 
     /**
+     * The inner types to go on this insertion. See {@link ReceiverInsertion}
```

Comment:
Are the insertions types (as the comment says) or possibly other things such as annotations and receiver declarations?